usagov/vote-gov-drupal
# VOTE-2885 configure state mail registration section

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2885

## Description

1. Add ability to hide state display content subfields from the form
2. Configure state mail registration section


## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/block/add/state_display_content and confirm that the 'Link text' subfield is hidden for Registration Intro field.
2. visit http://vote-gov.lndo.site/register/washington and confirm the content for mail and NVRF displays
3. visit http://vote-gov.lndo.site/register/wyoming and confirm the content for mail displays but not NVRF

## Checklist for the Developer




## Checklist for the Peer Reviewers
